content/news/2023-10-13-EGD-2023-the-wrapup/index.md

```diff
@@ -0,0 +1,141 @@
+ * [Exporting **RO-Crate and BCO** from Galaxy](https://galaxyproject.org/news/2023-02-23-structured-data-exports-ro-bco/) and [**publishing** from Galaxy via Invenio to Zenodo](https://github.com/galaxyproject/galaxy/pull/16381) ([WP2: Stimulate FAIR and reusable research](https://galaxyproject.org/news/2023-01-12-eurosciencegateway-wp2/))
+ * Update and extension of the **Pulsar network**; [new repository available](https://github.com/usegalaxy-eu/pulsar-deployment) and [development of a GA4GH TES-compliant API](https://github.com/CESNET/tesp-api) ([WP3: Distributed heterogeneous compute](https://galaxyproject.org/news/2023-01-19-eurosciencegateway-wp3/))
+ * Milestone on **Bring Your Own Compute/Storage** ([BYOC](https://drive.google.com/file/d/1I3FCJ8vmo6jESNqMcfGZHRcVxtk169Tm/view)/[BYOS](https://drive.google.com/file/d/1IUBbuowoCYsOAuaKzBtEtwayzY4oooQF/view)) achieved; principle adoption of [ARC as job runner](https://docs.google.com/presentation/d/1xb70Z4dZfXEJrvbXFhLQceA1g2UZnEWG) and efforts on [meta-scheduling](https://docs.google.com/presentation/d/1-5ch2UpOXH9qBouR6cbqBYbsgzSNMysk/edit#slide=id.p1)([WP4: Building blocks for a sustainable operating model](https://galaxyproject.org/news/2022-12-08-esg-wp4/))
+ * Lively **communities** and tools/workflows from e.g. [astronomy](https://galaxyproject.org/events/2023-10-egd/egd/Volodymyr_Savchenko-EGD-2023-astro.pdf), material sciences ([Muon Galaxy](https://galaxyproject.org/events/2023-10-egd/egd/Leandro-EGD-Muongalaxy-2023.pdf), [Catalysis workflows](https://galaxyproject.org/events/2023-10-egd/egd/Leandro-Applications-to-Catalysis-Workflows.pdf)) ([WP5: Community engagement, adoption and onboarding](https://galaxyproject.org/news/2022-11-24-esg-wp5/))
+ 
+Several results and work in progress were presented in the dedicated ["ESG meets EGD" session on Day 2](/events/2023-10-egd/egd/) as well as during the GA itself - [find all materials there](/events/2023-10-egd/esg/). Also check out our [ESG news page](/projects/esg/news/).
+
+### Numbers & some highlights
+
+* 3 full days
+* 77 registrations overall, by far onsite, and just two (!) badges not picked up
+* 41 ELIXIR members from 11 nodes
+* 28 ESG members from 15 partners
+* 25 public talks, 4 BOFs, 1 full-day CodeFest
+* Many breadroles, cookies and several units of coffee
+* Countless chats, connections, smiles
+
+An explicitly incomplete, unordered list of memorable impressions, and items worth to (re-)read yourself:
+
+* A delighted audience being served with a beautiful preview on [**'Workflow Comments'**](https://docs.google.com/presentation/d/1O20J7ifwagyW5X6VeK5hshqgfiehuNr39eTr2JA8u7U)
+* Having digitally met a large round of the **US team** (although 'owl-to-owl' talk remains subject to improvement...)
```

Comment:
"(although 'owl-to-owl' talk remains subject to improvement...)" ?? no reader knows what you mean